tests/Microsoft.Identity.Test.Unit/BrokerTests/WamTests.cs

```diff
@@ -603,6 +649,103 @@ public async Task RemoveAADAccountAsync()
             }
         }
 
+        [TestMethod]
+        public async Task GetAccounts_DoesNotCallPlugins_Async()
```

Comment:
Should this test method be **CallsPlugins** and next one **DoesNotCallPlugins**?